cmd/kubectl-gadget/seccompadvisor.go

```diff
@@ -0,0 +1,163 @@
+// // Copyright 2019-2021 The Inspektor Gadget authors
+// //
+// // Licensed under the Apache License, Version 2.0 (the "License");
+// // you may not use this file except in compliance with the License.
+// // You may obtain a copy of the License at
+// //
+// //     http://www.apache.org/licenses/LICENSE-2.0
+// //
+// // Unless required by applicable law or agreed to in writing, software
+// // distributed under the License is distributed on an "AS IS" BASIS,
+// // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// // See the License for the specific language governing permissions and
+// // limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/kinvolk/inspektor-gadget/cmd/kubectl-gadget/utils"
+	gadgetv1alpha1 "github.com/kinvolk/inspektor-gadget/pkg/api/v1alpha1"
+)
+
+var seccompAdvisorCmd = &cobra.Command{
+	Use:   "seccomp-advisor",
+	Short: "Generate seccomp policies based on recorded syscalls activity",
+}
+
+var seccompAdvisorMonitorCmd = &cobra.Command{
+	Use:   "monitor",
+	Short: "Monitor the system calls",
+	Run:   runSeccompAdvisorMonitor,
+}
+
+var seccompAdvisorReportCmd = &cobra.Command{
+	Use:   "report",
+	Short: "Report seccomp policies",
+	Run:   runSeccompAdvisorReport,
+}
+
+var seccompAdvisorListCmd = &cobra.Command{
+	Use:   "list",
+	Short: "List existing seccomp traces",
+	Run:   runSeccompAdvisorList,
+}
+
+var (
+	quiet            bool
+	externalResource string
+)
+
+func init() {
+	// Add generic information.
+	rootCmd.AddCommand(seccompAdvisorCmd)
+	utils.AddCommonFlags(seccompAdvisorCmd, &params)
+
+	seccompAdvisorCmd.AddCommand(seccompAdvisorMonitorCmd)
+	seccompAdvisorMonitorCmd.PersistentFlags().BoolVarP(&quiet, "quiet", "", false, "Only print traceID")
+	seccompAdvisorMonitorCmd.PersistentFlags().StringVarP(&externalResource, "external-resource", "", "", "ExternalResource name")
```

Comment:
`external-resource` it not clear here. What about `seccomp-profile`? 

I think external resource is an internal concept for the CRD, we should hide it from the CLI.